examples/AgraphExample.py

```diff
@@ -0,0 +1,73 @@
+# Ignoring some linting rules in tests
+# pylint: disable=redefined-outer-name
+# pylint: disable=missing-docstring
+import numpy as np
+
+from bingo.AGraph.AGraphCrossover import AGraphCrossover
+from bingo.AGraph.AgraphMutation import AGraphMutation
+from bingo.AGraph.AGraphGenerator import AGraphGenerator
+from bingo.AGraph.ComponentGenerator import ComponentGenerator
+from bingo.ExplicitRegression import ExplicitRegression, ExplicitTrainingData
+from bingo.EA.AgeFitnessEA import AgeFitnessEA
```

Comment:
`from bingo.EA.AgeFitnessEA import AgeFitnessEA` should be `bingo.Base.AgeFitnessEA import AgeFitnessEA`. However this still runs, possibly because of pycache? Is there a hidden EA folder still? Not sure how this wasn't caught.